# What happens when „1‟== 1 is executed ?

# It is Wrong Exception Handle in This Line